docs/adr/0001-mobile-app-architecture.md

```diff
@@ -0,0 +1,107 @@
+# Build Relay Mobile App with [Mobile Web | Qt | React Native | WebView ]
```

Comment:
Yeah, since we haven't made the decision yet, I included all of the options in the title. When we make our decision, we'll change this line to reflect that.